testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/suites/CoreResourceManagementTestCase.java

```diff
@@ -369,6 +389,50 @@ public void testSystemPropertyManagement() throws IOException {
         Assert.assertEquals(origPropCount, returnVal.asList().size());
     }
 
+    @Test
+    public void testSystemPropertyBootTime() throws IOException {
```

Comment:
I think for a test like this it would be good to use the core-service=platform-mbean stuff to validate the actual system property in the server VM. org.jboss.as.test.deployment.trivial.ServiceActivatorDeploymentUtil has some utility code related to reading system properties that way.